CHANGELOG.unreleased.md

```diff
@@ -12,6 +12,7 @@
 - [Home] Always sort the items by their names as a secondary sort criteria [#3983](https://github.com/vatesfr/xen-orchestra/issues/3983) (PR [#4047](https://github.com/vatesfr/xen-orchestra/pull/4047))
 - [Remotes] Fixes `spawn mount EMFILE` error during backup
 - Properly redirect to sign in page instead of being stuck in a refresh loop
+- [Backup-ng] Properly see the matching VMs corresponding to the specified tags [#4078](https://github.com/vatesfr/xen-orchestra/issues/4078) (PR [#4085](https://github.com/vatesfr/xen-orchestra/pull/4085))
```

Comment:
As we discussed, the issue only appears when clicking the link, on the Home page. I think we should make it clearer.